src/main/scala/org/neo4j/spark/util/Validations.scala

```diff
@@ -129,4 +131,13 @@ object Validations {
       cache.close()
     }
   }
+
+  def validateConnection(session: Session): Unit = {
+    try {
+      session.run("EXPLAIN RETURN 1").consume()
```

Comment:
Yep, that'd work.  Honestly you don't even need the explain, you could just roundtrip "something".   `RETURN 1` by itself would work, because it proves bi-directional communication and that the session is sensible, but either way is really fine, since you're not doing anything with the explain anyway